README.md

```diff
@@ -26,6 +26,14 @@ Usage (as script in `package.json`):
 
 ## Options
 
+### Java runtime arguments
+
+You can pass options to Java runtime with `--java-arg` CLI argument:
+
+```bash
+wiremock --java-arg -Dmy.custom.var=some-value --root-dir ./mock
```

Comment:
I think this may not work as there's no --permitted-system-keys argument permitting the use of the variable my.custom.var. If you have {{ my.custom.var }} in a response mapping, what you will see when the mappings requested in it's place is [ERROR: Access to my.custom.var is denied] 

It's probably a bit of work but if you could build up a comma seperated list of the -D options and pass it to a constructed --permitted-system-keys argument that would be the ideal solution. Or else mention somewhere that only -D options that match wiremock.* are allowed by default.